edl/sgx/platform.edl

```diff
@@ -18,4 +18,5 @@ enclave
     from "cpu.edl" import *;
     from "debug.edl" import *;
     from "thread.edl" import *;
+    from "switchless.edl" import *;
```

Comment:
Now that switchless is part of platform.edl for sgx, remove the switchless.edl wrapper